kratos.gid/kratos_default.spd

```diff
@@ -8,12 +8,12 @@
 	<hiddenfield n="activeapp" active="0" v="" state="disabled"/>
 	<value n="nDim" pn="Dimension" un="nDim" v="undefined" state="disabled" values="undefined,2D,2Da,3D" dict="undefined,Undefined,2D,2D,2Da,2D axisymmetric,3D,3D"/>
 	<appLink n="Solid" public="1" pn="Solid" active="0" appid="Solid" prefix="SL"/>
+	<appLink n="Pfem" public="1" pn="Pfem" active="0" appid="Pfem" prefix="PFEM_"/>

```

Comment:
There is no need for this change

Reply:
I put the two applications that I manage together. If you prefer you can move them whenever you want.

Reply:
I have restored the position of the Pfem app